js/a11y.js

```diff
@@ -163,32 +164,66 @@ class A11y extends Backbone.Controller {
 
   /**
    * Calculate the aria level for a heading
-   * @param {string|number} [defaultLevelOrType] Specify a default level or type group.
-   * @param {string|number} [overrideLevelOrType] Override with a level or type group from the designer.
+   * @param {object} [options]
+   * @param {string|number} [options.id] Used to automate the heading level when relative increments are used
+   * @param {string|number} [options.level] Specify a default level, "component" / "menu" / "componentItem" etc
+   * @param {string|number} [options.override] Override with a default level, an absolute value or a relative increment
    * @returns {number}
+   * @notes
+   * Default levels come from config.json:_accessibility._ariaLevels attribute names, they are:
+   *  "menu", "menuGroup", "menuItem", "page", "article", "block", "component", "componentItem" and "notify"
+   * An absolute value would be "1" or "2" etc.
+   * A relative increment would be "@page+1" or "@block+1". They are calculated from ancestor values,
+   * respecting both _ariaLevel overrides and not incrementing for missing displayTitle values.
    */
-  ariaLevel(defaultLevelOrType = 1, overrideLevelOrType) {
-    // get the global configuration from config.json
-    const cfg = Adapt.config.get('_accessibility');
-
-    // first check to see if the Handlebars context has an override
-    if (overrideLevelOrType) {
-      defaultLevelOrType = overrideLevelOrType;
+  ariaLevel({
+    id = null,
+    level = "1",
+    override = null
+  } = {}) {
+    if (arguments.length > 1) {
```

Comment:
Shouldn't this be 
```suggestion
    if (arguments.length === 2) {
```
if we are trying to catch old implementations?

Reply:
New implementations will have only one argument. Either should be fine?